### PR TITLE
#169257416 Fix the Profile and Comment tests bug

### PR DESCRIPTION
**Pull request by gildniy**
#### What does this PR do?
Fixes the Profile and Comment tests bug, which was causing to other tests to fail.
#### Description of Task to be completed?
- Wrapped the `before` hook of the test inside the `describe` section
#### How should this be manually tested?
- Clone the repo to the local machine
- Run `npm run pretest-travis` to migrate required tables
- Run `npm run test`
All test will run well as expected.
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: src/tests/comments.test.js [hunk 1]
```diff
@@ -6,83 +6,84 @@ import users from './mock-data/test-users';
...
 import db from '../models';
+import truncate from './scripts/truncate';
 
 use(chaiHttp);
 
-const prefix = '/api/v1';
-let rightUserToken, wrongUserToken, requestId, commentExample;
+describe('', () => {
+  const prefix = '/api/v1';
+  let rightUserToken, wrongUserToken, requestId, commentExample;
```
Comment (b0nbon1) [inline src/tests/comments.test.js]: Nice work, but I think the `describe` name is missing

**Review by b0nbon1 — CHANGES_REQUESTED** [review]

File: src/tests/comments.test.js (hunk 1 above)
Comment (gildniy) [reply src/tests/comments.test.js]: > Nice work, but I think the `describe` name is missing

Ah ok, let me add a description for the file.

Comment (gildniy) [reply src/tests/comments.test.js]: Now fixed, thanks @b0nbon1

**Review by b0nbon1 — APPROVED** [review]
It looks good now. I think it is ready to be mergedd

**Review by timolinn — APPROVED** [review]

**Review by nshaphan — APPROVED** [review]
This fixes the issue

**Review by collinewait — APPROVED** [review]

**Pull request merged**